loopchain/blockchain/blockchain.py

```diff
@@ -296,6 +298,26 @@ def __remove_block_up_to_target(self, target_block: 'Block'):
     def rollback(self, target_block):
         self.__remove_block_up_to_target(target_block)
 
+    def get_synced_block_height(self):
+        try:  # If blockchain_store from backup.
+            synced_block_height_bytes = self._blockchain_store.get(BlockChain.SYNCED_BLOCK_HEIGHT)
+            synced_block_height = int.from_bytes(synced_block_height_bytes, byteorder='big')
+        except KeyError:  # Origin blockchain_store.
+            synced_block_height = self.__last_block.header.height
+        return synced_block_height
+
+    def get_block(self, block_hash, block_height):
+        try:
+            block_dump = self._blockchain_store.get(block_hash.encode(encoding='UTF-8'))
+            block_version = self.__block_versioner.get_version(block_height)
+            block_serializer = BlockSerializer.new(block_version, self.__tx_versioner)
+            block = block_serializer.deserialize(json.loads(block_dump))
+        except KeyError:
+            block = self.__block_manager.block_request_by_citizen(block_height)
```

Comment:
How about editing the method name from `by` to `as` if `block_request_by_citizen` means that this node requests a block as a citizen? If or not, skip this comment, please.